web/packages/teleport/src/Discover/ConnectMyComputer/SetupConnect/SetupConnect.tsx

```diff
@@ -213,16 +215,35 @@ export function SetupConnect(
       </StyledBox>
 
       <StyledBox>
-        <Text bold>Step 2: Sign In and Connect My Computer</Text>
+        <Flex gap={5} flexWrap="wrap">
```

Comment:
`flex-wrap` doesn't have any real effect here since Discover has fixed width there. It's more for future proofing, in case that gets ever changed.